poly-synth/src/poly-synth.js

```diff
@@ -43,8 +43,8 @@ class PolySynth {
    * @param {Number} frequency the corresponding frequency of the note, e.g 440
    */
   playNote(note, frequency) {
-    let voice =
-        new Voice(this.context_, frequency, this.lowPassCutoff_, this, {});
+    let voice = new PolySynthVoice(
```

Comment:
How about this?

In `playNote()`:
```js
let voice = new PolySynthVoice(this, pitch, this.getParameters());
this.activeVoices_.set(pitch, voice); // activeVoices_ is Map()
```
Use `pitch` when possible, so you can easily modulate with detune or offset. For example, two oscillators with different detunes can be played by the same pitch. The `synth` handling the high-level stuff (pitch), whereas all the details (freq) can be handled by each voice.

In the constructor of `PolySynthVoice`:
```js
constructor (synth, pitch, parameters) {
  this.synth_ = synth;
  /* then do something with parameters (cutoff, Q, ADSR) */
  this.start();
}
```